src/main/docker/Dockerfile.jre

```diff
@@ -24,9 +43,15 @@ ADD https://raw.githubusercontent.com/CAFapi/caf-common/v1.14.0/container-cert-s
     /startup/startup.d/
 RUN chmod +x /startup/startup.d/install-ca-cert-java.sh
    
-# Install OpenJDK 8 Runtime Environment
+# Install OpenJDK 8 Runtime Environment and Patch
 RUN zypper -n install java-1_8_0-openjdk
 
+# Set Java Home
+ENV JAVA_HOME=/usr/lib64/jvm/java-1.8.0-openjdk-1.8.0
```

Comment:
Is it right that this is being added?

Reply:
It is only used in the build to shorten the path for the patch command and the copy command but I don't think it does any harm and think it is weird it was not there already.

Reply:
As discussed, officially it is for JDK:
https://docs.oracle.com/javase/8/docs/technotes/guides/troubleshoot/envvars001.html